### ISSUE TITLE
Ipelets don't build

### ISSUE THREAD

**Issue by mrzv** [CGAL/cgal#588]
I'm trying to build Ipelets from CGAL 4.7 with Ipe 7.2.1. I'm using Boost 1.60. I configure and build them using:
```
  cd "demo/CGAL_ipelets"
  cmake . -DCMAKE_CXX_FLAGS="-std=c++11" -DWITH_IPE_7=ON
  make
```

Most things build, but eventually I get the following error:
```
Scanning dependencies of target CGAL_svdlinf
[ 80%] Building CXX object CMakeFiles/CGAL_svdlinf.dir/svdlinf.cpp.o
In file included from /usr/include/boost/iterator/iterator_adaptor.hpp:10:0,
                 from /usr/include/boost/iterator/transform_iterator.hpp:12,
                 from /usr/include/CGAL/boost/iterator/transform_iterator.hpp:35,
                 from /usr/include/CGAL/Coercion_traits.h:35,
                 from /usr/include/CGAL/number_type_basic.h:43,
                 from /usr/include/CGAL/basic.h:44,
                 from /usr/include/CGAL/Cartesian/Cartesian_base.h:28,
                 from /usr/include/CGAL/Simple_cartesian.h:28,
                 from /usr/include/CGAL/Exact_predicates_inexact_constructions_kernel.h:28,
                 from /tmp/makepkg/cgal-ipelets/src/CGAL-4.7/demo/CGAL_ipelets/skeleton.cpp:21:
/usr/include/boost/graph/adjacency_matrix.hpp:446:5: error: ‘ice_not’ is not a member of ‘boost::type_traits’
     BOOST_STATIC_ASSERT(type_traits::ice_not<(is_same<Directed, bidirectionalS>::value)>::value);
     ^
/usr/include/boost/graph/adjacency_matrix.hpp:446:5: error: ‘::value’ has not been declared
     BOOST_STATIC_ASSERT(type_traits::ice_not<(is_same<Directed, bidirectionalS>::value)>::value);
     ^
/usr/include/boost/graph/adjacency_matrix.hpp:446:5: note: suggested alternative:
In file included from /usr/include/boost/bind.hpp:22:0,
                 from /usr/include/CGAL/Straight_skeleton_2/Straight_skeleton_builder_2_impl.h:22,
                 from /usr/include/CGAL/Straight_skeleton_builder_2.h:930,
                 from /usr/include/CGAL/create_straight_skeleton_2.h:23,
                 from /usr/include/CGAL/create_offset_polygons_2.h:23,
                 from /tmp/makepkg/cgal-ipelets/src/CGAL-4.7/demo/CGAL_ipelets/skeleton.cpp:24:
/usr/include/boost/bind/bind.hpp:118:25: note:   ‘boost::_bi::value’
 template<class T> class value
                         ^
```

**Comment by sloriot** [comment]
I think it is a bug of boost 1.60
Could you try an earlier version?

**Comment by mrzv** [comment]
I know that ipelets from 4.6 were building fine with 1.59. It's difficult for me to try 4.7 with 1.59, but I agree, the problem looks to be upstream. But I'm not sure what the best way to resolve it is.

**Comment by sloriot** [comment]
Fixed by 83f10ef3
